packages/light-client-extension-helpers/README.md

```diff
@@ -211,4 +211,45 @@ client.supervise()
 
 ### tx-helper
 
-TODO: write usage guide
+The tx-helper package allows you to easily sign transactions. You just need to calldata and an implementation of the
```

Comment:
Should this be `You just need calldata`?

Reply:
no u need a signer implementation too

Reply:
for example: https://github.com/polkadot-js/extension/pull/1389/files#diff-03379f3e1169c9b022a9b6ece3a998003b83b9f9c1417243026c8ef722c23902R78

Reply:
Oh yea, I understand that. Im asking should the sentence read `You just need calldata...` as opposed to `You just need to calldata...`

Reply:
oh oops :)